it/AssetsControllerISpec.scala

```diff
@@ -14,38 +14,76 @@
 
 import scala.collection.JavaConverters._
+import scala.io.Source
+import scala.util.Try
+import scala.xml.XML
 
 class AssetsControllerISpec extends IntegrationSpecBase {
 
-  val wsdlBaseUrl = s"http://localhost:$port/assets/wsdl/"
-
+  val wsdlBaseUrl = s"http://localhost:$port/assets/wsdl/eu/outbound/CR-for-NES-Services/"
   val wsdlOperationsForFileNames = Map(
-    "CCN2.Service.Customs.Default.ICS.ReferralManagementBAS_1.0.0_CCN2_1.0.0.wsdl" -> List(
+    "BusinessActivityService/ICS/ReferralManagementBAS/V1/CCN2.Service.Customs.Default.ICS.ReferralManagementBAS_1.0.0_CCN2_1.0.0.wsdl" -> List(
       "IE4Q04requestAdditionalInformation",
       "IE4R02provideAdditionalInformation",
       "IE4Q05requestHRCM",
       "IE4R03provideHRCMResult"
     ),
-    "CCN2.Service.Customs.Default.ICS.RiskAnalysisOrchestrationBAS_1.0.0_CCN2_1.0.0.wsdl" -> List(
+    "BusinessActivityService/ICS/RiskAnalysisOrchestrationBAS/V1/CCN2.Service.Customs.Default.ICS.RiskAnalysisOrchestrationBAS_1.0.0_CCN2_1.0.0.wsdl" -> List(
       "IE4N03notifyERiskAnalysisHit",
       "IE4S01submitEScreeningAssessment",
       "IE4S02submitRiskAnalysisResult",
       "IE4S02updateERiskAnalysisResult",
       "IE4S01updateEScreeningResult"
-    )
+    ),
+    "BusinessActivityService/ICS/RiskAnalysisOrchestrationBAS/V1/CCN2.Service.Customs.Default.ICS.RiskAnalysisOrchestrationBAS_1.0.0_1.0.0.wsdl" -> List.empty,
+    "BusinessActivityService/ICS/ReferralManagementBAS/V1/CCN2.Service.Customs.Default.ICS.ReferralManagementBAS_1.0.0_CCN2_1.0.0.wsdl" -> List.empty
   )
 
+  "all EU Files within public folder" should {
+    val xmlSchemaExtension = ".xsd"
+    val xmlWsdlExtension = ".wsdl"
+    val baseDirectory = new File(app.path.getCanonicalPath + "/public/wsdl/eu/outbound/CR-for-NES-Services" )
+    val allFilesFromEU = recursiveListFiles(baseDirectory).filter(_.isFile).filterNot(_.isHidden)
+
+    "be correct amount of xsds and wsdls" in {
+      allFilesFromEU.count(file => file.getName.contains(xmlSchemaExtension) || file.getName.contains(xmlWsdlExtension)) shouldBe 65
+    }
+
+    "not contain {DestinationID} as this should have been replaced" in {
+      allFilesFromEU.exists(_.getName.contains("{DestinationID}")) shouldBe false
+    }
+
+    s"return ${Status.OK} and parse to xml if $xmlSchemaExtension / $xmlWsdlExtension" when {
+      allFilesFromEU.foreach( eachFile =>
+        s"file is ${eachFile.getName}" in {
+          val pathToFile = eachFile.getCanonicalPath.split("/CR-for-NES-Services/")(1).trim
+          val resultOfGettingAsset = await(buildClient(s"/assets/wsdl/eu/outbound/CR-for-NES-Services/$pathToFile").get())
+          resultOfGettingAsset.status shouldBe Status.OK
+
+          if(pathToFile.contains(xmlSchemaExtension) || pathToFile.contains(xmlWsdlExtension)) {
+            val fileFromDirectoryParsed = {
+              Try(XML.load(new ByteArrayInputStream(Source.fromFile(eachFile,"UTF-8").mkString.getBytes("UTF-8"))))
```

Comment:
Probably not a big deal but you can close your source like this:
```suggestion
              val source = Source.fromFile(eachFile, "UTF-8")
              try Try(XML.load(new ByteArrayInputStream(source.mkString.getBytes("UTF-8"))))
               finally source.close
```